libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.cpp

```diff
@@ -168,23 +168,23 @@ const AP_Param::GroupInfo AC_AttitudeControl_Multi::var_info[] = {
     // @Param: RAT_YAW_FLTT
     // @DisplayName: Yaw axis rate controller target frequency in Hz
     // @Description: Yaw axis rate controller target frequency in Hz
-    // @Range: 1 5
+    // @Range: 1 50
     // @Increment: 1
     // @Units: Hz
     // @User: Standard
 
     // @Param: RAT_YAW_FLTE
     // @DisplayName: Yaw axis rate controller error frequency in Hz
     // @Description: Yaw axis rate controller error frequency in Hz
-    // @Range: 1 10
+    // @Range: 0 20
     // @Increment: 1
     // @Units: Hz
     // @User: Standard
 
     // @Param: RAT_YAW_FLTD
     // @DisplayName: Yaw axis rate controller derivative frequency in Hz
     // @Description: Yaw axis rate controller derivative frequency in Hz
-    // @Range: 1 20
+    // @Range: 5 100
```

Comment:
I missed this. I think this is too high for yaw. I would make it the same as FLTT - so 50, unless @lthall disagrees

Reply:
@andyp1per, np, reduced ATC_RAT_YAW_FLTD's upper limit to 50